rbe.bazelrc

```diff
@@ -0,0 +1,71 @@
+
+# Copyright 2016 The Bazel Authors. All rights reserved.
```

Comment:
2019
Also please assign the copyright to the correct owner (Kubernetes)

Reply:
Actually just drop the copyright stanza, we don't need it for config.

Reply:
Please address this when you get the chance, then we can merge (drop the copyright)

Reply:
Done. Removed the copyright stanza.